products/tarantool.md

```diff
@@ -0,0 +1,68 @@
+---
+permalink: /tarantool
+layout: post
+title: Tarantool
+command: $ tarantool --version
+releasePolicyLink: https://www.tarantool.io/en/doc/latest/release/policy/
+changelogTemplate: |
+  https://www.tarantool.io/en/doc/latest/release/__LATEST__/
+category: db
+eolColumn: Support Status
+releaseDateColumn: false
+sortReleasesBy: 'releaseCycle'
+releases:
+  - releaseCycle: "2.8"
+    release: 2021-12-22
+    eol: false
+    latest: "2.8.3"
+
+  - releaseCycle: "2.7"
+    release: 2021-08-19
+    eol: 2021-08-19
+    latest: "2.7.3"
+
+  - releaseCycle: "2.6"
+    release: 2021-04-21
+    eol: 2021-04-21
+    latest: "2.6.3"
+
+  - releaseCycle: "2.5"
+    release: 2020-12-30
+    eol: 2020-12-30
+    latest: "2.5.3"
+
+  - releaseCycle: "2.4"
+    release: 2020-10-22
+    eol: 2020-10-22
+    latest: "2.4.3"
+
+  - releaseCycle: "2.3"
+    release: 2020-07-17
+    eol: 2020-07-17
+    latest: "2.3.3"
+
+  - releaseCycle: "2.2"
+    release: 2019-12-31
+    eol: 2020-04-20
+    latest: "2.2.2"
+
+  - releaseCycle: "1.10"
+    release: 2021-12-22
+    eol: false
+    latest: "1.10.12"
+
+---
+
+> [Tarantool](https://www.tarantool.io/) is an in-memory computing platform consisting of a database and an application server. It is distributed under BSD 2-Clause terms. Key features of the application server: heavily optimized Lua interpreter with incredibly fast tracing JIT compiler, based on LuaJIT 2.1, cooperative multitasking and non-blocking IO, persistent queues, sharding, cluster and application management framework, access to external databases such as MySQL and PostgreSQL, a rich set of built-in and standalone modules.
+
+The Tarantool release policy is changing to become more clear and intuitive. The new policy uses a [SemVer-like](https://semver.org/) versioning format, and introduces a new version lifecycle with more long-time support series. This document explains the new release policy, versioning rules, and [release series](https://www.tarantool.io/en/doc/latest/release/policy/#term-Release-series) lifecycle.
```

Comment:
I'm not sure if talking about the existence of 2 release policies is helpful or confusing, without providing important details about the differences between these policies (such as what releases are stable and LTS, and how long the support periods are).

Reply:
Thank you for comments!

In the old policy we had X.Y series with X.Y.1 as beta, X.Y.2 as stable release and X.Y.3 as the second and the last stable release. We release a series roughly once per quarter, so the support period was too short: 3 months since the first stable till the last release.

LTS is the exception here: we don't stop on X.Y.3, but continue roughly per quarter releases.

Aside of other changes the new release policy holds our intention to support a series for at least two years since a first release. We don't use the 'LTS' term in the new policy, but in fact all series become LTS.

More in-depth document about the new release policy is [here](https://github.com/tarantool/tarantool/discussions/6182).

----

To be honest, I don't know how to present the information better. We have no releases by the new policy ATM (only two beta versions) and it may be a source of a confusion. However we'll have 2.10.0 this year and, it seems, a summary of differences would be helpful.

Can (and should?) we add pre-releases (2.10.0-beta1 and 2.10.0-beta2) into the table to make things more clear? We can remove them after a release.

If you have other thoughts and/or suggestion how to better present the information, I would be very grateful.

Reply:
Let me try to file a PR against this with some suggestions 👍🏽

Reply:
Okay. Let me know about your variant of wording. I'm one of authors of the new policy and would like to see how to better present it and would like to check facts.

Reply:
@captn3m0 ?

Reply:
Merging this, will file a separate PR when I can, will tag you for a review there. Is the https://www.tarantool.io/en/doc/latest/release/legacy-policy/ maintained on GitHub? I could file a PR there as well perhaps?

Reply:
> Is the https://www.tarantool.io/en/doc/latest/release/legacy-policy/ maintained on GitHub? I could file a PR there as well perhaps?

Yep. The source is here: https://github.com/tarantool/doc/blob/latest/doc/release/legacy-policy.rst.